commons/src/test/java/org/sonarsource/analyzer/commons/RuleMetadataLoaderTest.java

```diff
@@ -500,6 +501,19 @@ class TestRule {
       .containsExactlyInAnyOrder("cwe:311", "cwe:315", "cwe:614", "owaspTop10:a2", "owaspTop10:a3", "owaspTop10-2021:a4", "owaspTop10-2021:a5");
   }
 
+  @Test
+  public void test_security_standards_on_10_10_return_stig() {
```

Comment:
should we have a test for before 10.10?

Reply:
I wanted to do it, but it's already covered by the test below (`test_security_standards_on_9_9_return_asvs`). It reads the same file and checks the same expectations that we we would have had on 9.9. Or would you like to have a test for let's say 10.9?

Reply:
ahh I see! I think that's good then!